### PR TITLE
Cleanup Unit Tests

### PR DESCRIPTION
**Pull request by talekhinezh**

### CODE REVIEW

File: radix-engine/tests/test_runner.rs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+use radix_engine::model::Receipt;
+use radix_engine::model::Transaction;
+
+pub struct TestRunner<'l> {
+    executor: TransactionExecutor<'l, InMemorySubstateStore>,
+}
+
+impl<'l> TestRunner<'l> {
+    pub fn new(ledger: &'l mut InMemorySubstateStore) -> Self {
+        let executor = TransactionExecutor::new(ledger, true);
```
Comment (iamyulong) [inline radix-engine/tests/test_runner.rs:13]: Minor: I'd suggest we make `TestRunner` own an `InMemorySubstateStore`, and allow the constructor to take a parameter `trace` (I personally found it useful when writing tests).

**Review by iamyulong — APPROVED** [review]
LGTM

**Comment by iamyulong** [comment]
Looks like `rustfmt` is having problem with resolving modules under `tests`. Script `./format.sh`  stops working.

**Comment by iamyulong** [comment]
A temporary workaround would be
```
#[rustfmt::skip]
mod test_runner;
```

File: radix-engine/tests/test_runner.rs (hunk 1 above)
Comment (talekhinezh) [reply radix-engine/tests/test_runner.rs:13]: Was looking to do this initially but then ran into awkwardness with certain borrowed structures. Will probably fix this in a later refactor.

**Pull request merged**